reference_data/management/commands/update_gencode.py

```diff
@@ -94,7 +94,7 @@ def update_gencode(gencode_release, gencode_gtf_path=None, genome_version=None,
         coding_region_size_field_name = "coding_region_size_grch{}".format(genome_version)
 
         logger.info("Loading {} (genome version: {})".format(gencode_gtf_path, genome_version))
-        with gzip.open(gencode_gtf_path) as gencode_file:
+        with gzip.open(gencode_gtf_path, 'rt') as gencode_file:
```

Comment:
Adding 'rt' is necessary because it will read out `binary` data in Python 3 and binary data will cause errors in following string operations.